BLSQ/iaso
# IA-2776 change request approved comment

In the below GIF the first use case we accept partially the modifications, but it would be great to be able to send out a comment to warn the mobile app user that he needs to re-submit with the missing part? …
![demopoliogeoregistry](https://github.com/BLSQ/iaso/assets/12494624/0738f8c6-1fc1-4afc-aba2-b59036ed16e1)

Related JIRA tickets : IA-2776
## Self proofreading checklist



## Changes

- fix group field color while approved, it was always green even if change was not applied
- rename comment dialog
- allow to display comment dialog if partially approved
- adapt backend to save comment

## How to test

You need a new org unit change request with multiple changes.
Open the dialog to approve or reject, select only one field and click on approve selected changes, you should be able to enter a a comment.
Save and check that comment has been saved

## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/bd1a4a96-bfa0-4b51-bf56-cac4c75a202e



## Notes

We should check that mobile app is displaying also comment if change request is partially approved @bmonjoie
